ngraph_bridge/ngraph_builder.cc

```diff
@@ -5056,6 +5136,27 @@ Status Builder::TranslateGraph(
     result->set_needs_default_layout(true);
   }
 
+  // TODO: Use is_type<ng::op::Result>(n) when we upgrade to new ngraph
+  auto check_if_result_type = [&ng_function](shared_ptr<ng::Node> n) {
```

Comment:
Are we saying ng::op::Result dont need a provenance tag?

Reply:
Yes.. Result and Parameter don't have provenance tags.